### PR TITLE
sonic linux: added template for 'portstat'

### PR DESCRIPTION
**Pull request by sidtyagi**
##### ISSUE TYPE
 - New Template Pull Request

##### COMPONENT

sonic_linux_portstat.template, linux, portstat

### CODE REVIEW

File: ntc_templates/__init__.py [hunk 1]
```diff
@@ -1,3 +1,3 @@
 """ntc_templates - Parse raw output from network devices and return structured data."""
 
-__version__ = '1.0.40.dev1'
+__version__ = '1.0.41.dev1'
```
Comment (rvisnu) [inline ntc_templates/__init__.py]: Version is changed. please check

Comment (sidtyagi) [reply ntc_templates/__init__.py]: done

File: templates/index [hunk 2]
```diff
@@ -272,7 +272,7 @@ sonic_linux_uptime_since.template, .*, linux, up[[time]] -s
 sonic_linux_free_dash_m.template, .*, linux, free -m
 sonic_linux_ps_dash_ef.template, .*, linux, ps -ef
 sonic_linux_df_dash_h.template, .*, linux, df -h
-
+sonic_linux_portstat.template, .*, linux, portstat
```
Comment (Sandeepsr) [inline templates/index]: can you arrange by length of line

Comment (sidtyagi) [reply templates/index]: done

**Pull request merged**